templates/activity/parts/activity-consumption.hbs

```diff
@@ -46,6 +46,11 @@
         {{else}}
         <div class="empty">{{ localize "DND5E.None" }}</div>
         {{/each}}
+
+        {{#if showConsumeSpellSlot}}
```

Comment:
Let's move this to above the consumption target parts. This will match what we do with things like damage parts and summon profiles.